### PR TITLE
Add reviewers

### PR DESCRIPTION
**Pull request by jmaspons**
For https://github.com/ropensci/software-review/issues/633

### CODE REVIEW

**Comment by jmaspons** [comment]
@jonthegeek, @ccamara can you confirm that's find and propose change if needed (e.g., orcid or whatever)?

File: DESCRIPTION [hunk 1]
```diff
@@ -2,9 +2,12 @@ Type: Package
 Package: osmapiR
 Title: OpenStreetMap API
 Version: 0.0.0.40
-Authors@R: 
+Authors@R: c(
     person("Joan", "Maspons", , "joanmaspons@gmail.com", role = c("aut", "cre"),
-           comment = c(ORCID = "0000-0003-2286-8727"))
+           comment = c(ORCID = "0000-0003-2286-8727")),
+    person("Jon", "Harmon", role = "rev"),
```
Comment (jonthegeek) [inline DESCRIPTION:8]:
```suggestion
    person("Jon", "Harmon", role = "rev", comment = c(ORCID = "0000-0003-4781-4346")),
```

**Review by jonthegeek — APPROVED** [review]
I added my ORCID. Merge that in and then it looks good to me!

**Comment by jmaspons** [comment]
Merging now. Feel free to ask for changes, @ccamara

**Comment by ccamara** [comment]
Thanks for this, @jmaspons , just submitted a PR to include my orcid: https://github.com/jmaspons/osmapiR/pull/33

**Pull request merged**